src/src/cron/faFaceIndexing.cron.ts

```diff
@@ -200,7 +200,7 @@ export class FaFaceIndexingCron {
 		// We need to skip processing records wch were processed in past X hours
 		let dt = new Date();
 		let filterTimestamp = new Date(
-			dt.setHours(dt.getHours() - 4),
+			dt.setHours(dt.getHours() - 2),
```

Comment:
Get this `2` from config / env

Reply:
what is 2 for?

Reply:
2 is hours limit for last_processed. just like 4 we used in attedance face rekognition.
after testing done on dev I will move it in env varibale

Reply:
fixed this